CenterForOpenScience/osf.io
# [ENG-2124] Adding storage caps funcitonality to node page in admin app

## Purpose

Admins need to be able to storage usage caps on nodes. This includes: 
1. Seeing current usage.
2. Changing storage caps for the node
3. Settings a public node to private regardless of whether it breaks storage caps.
4. Manually recalculating the storage usage of a node.

## Changes

Adding modal dialogues for several of the above functionality requirements.
Adding views for the modals.

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify you can accurately see a node's storage usage
- Verify you can update a nodes storage caps
- Verify you can make a node private, even if that would push the node over private storage cap.
- Verify you can manually recalculate the storage usage of a node

What are the areas of risk?
Shouldn't be any.

Any concerns/considerations/questions that development raised?
N/A

## Documentation

N/A

## Side Effects

Shouldn't be
## Ticket

https://openscience.atlassian.net/browse/ENG-2124
